recipes/taskflow/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  2.2.0:
```

Comment:
Do you really need 2.2.0 and 2.4.0? Usually we start providing only the latest version.

Reply:
This is a not a new library, this will replace cpp-taskflow` which got renamed. See the related issue.